recipes-support/libnl/libnl_%.bbappend

```diff
@@ -0,0 +1,11 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI_append = " file://FindNL.cmake \
+                   file://FindLibNL.cmake \
+                 "
+
+do_install_append() {
+    install -d ${D}${datadir}/cmake/Modules
+    # install -m 0644 ${WORKDIR}/FindNL.cmake ${D}${datadir}/cmake/Modules
+    install -m 0644 ${WORKDIR}/FindLibNL.cmake ${D}${datadir}/cmake/Modules
```

Comment:
What the reason to add two cmake modules manually and install only one of them? Why this modules added to libnl? Libnl already provided pkg-config that can define all variables for all build systems

Reply:
In addition, it's not going to be used by anything... prplMesh does `find_package(nl-3)` so it will load a file called `Findnl-3.cmake`, not `FindNL.cmake`.

This commit really should be split into smaller commits (that preferably still work with current OE) where each commit message explain what problem is solved.

Reply:
This bbappend has been removed
`prplmesh.bb` started use `*.cmake` files from prplMesh sources tree